src/modules/mod.ts

```diff
@@ -1,16 +1,19 @@
-import { default as CookiecordClient, Module, listener } from 'cookiecord';
-import { Message } from 'discord.js';
+import { Module, listener } from 'cookiecord';
+import { Message, GuildMember } from 'discord.js';
 import { rulesChannelId } from '../env';
 
 // Most job posts are in this format:
 // > [FOR HIRE][REMOTE][SOMETHING ELSE]
 // > Hi, I'm ponyman6000. Hire me!
 const jobPostRegex = /^(?:\[[A-Z ]+\]){2,}\n/i;
 
+// If more than RAID_JOINS occur in RAID_SECONDS, we're probably getting raided.
+const RAID_JOINS = 4;
+const RAID_SECONDS = 7;
```

Comment:
i feel this *might* be a little bit sensitive

Reply:
This is what Vortex is currently set to use

Reply:
i see 👍

Reply:
I feel like maybe this could be relative to normal traffic instead of absolute numbers? Might be a bit overkill though.

Reply:
i wonder what the current joins look like? then probably tweak these numbers to fit the needs of the server specifically